CHANGELOG.rst

```diff
@@ -1,21 +1,10 @@
-Change Log
-##########
+Changelog
+#########
 
-..
-   All enhancements and patches to codejail_service will be documented
-   in this file.  It adheres to the structure of https://keepachangelog.com/ ,
```

Comment:
You are still using:
> It adheres to the structure of https://keepachangelog.com/

Reply:
Oh! I thought I deleted that part.

Reply:
Oh I see what you're saying -- that line is still relevant and _shouldn't_ be removed.

Reply:
You did. I'm saying maybe you shouldn't, since you are still using "Added", "Fixed", etc.

Reply:
Yup. Restored that section.